default-templates/new-account/public/.acl

```diff
@@ -6,6 +6,9 @@
 <#owner>
     a acl:Authorization;
     acl:agent <{{webId}}>;
+{{#if addOwlSameAs}}
```

Comment:
If you Register new account with externalWebID from Delegator, 
the code from **{{#if addOwlSameAs}}** is added. 
If you Register new account with externalWebID from TLS Certificate, it isn't added.
p.s.
It is done, because I could not add owl:sameAs relation for externalWebID(that is the Delegator also). **owl:sameAs** isn't supported by rdflib.js

Reply:
I added the next relation to **card** profile (if externalWebID is from Delegator)
```
:me  owl:sameAs <${externalWebID}> .
```